dxr/static/templates/partial/promoted.html

```diff
@@ -0,0 +1,37 @@
+{% from "partial/result_lines.html" import result_lines -%}
+
+{% macro promoted_list(query, promoted, promoted_count, www_root, tree) -%}
+  {% for result in promoted %}
+    <div class="promoted-row">
+      <div class="promoted-icon {{ result.iconClass }}"></div>
+      <div class="promoted-result">
+        <span class="promoted-path">{{ result.pathLine | safe }}</span>
+        {% for entry in result.lines %}
+          <div class="promoted-line">
+            <span class="left-column">
+             <a href="{{ www_root }}/{{ tree }}/source/{{ result.path }}#{{ entry.line_number }}">
+               {{ entry.line_number }}
+             </a>
+           </span>
+           <span class="promoted-text">
+             <a href="{{ www_root }}/{{ tree }}/source/{{ result.path }}#{{ entry.line_number }}">
+               <code aria-labelledby="{{ entry.line_number }}">{{ entry.line|safe }}</code>
+             </a>
+           </span>
+          </div>
+        {% endfor %}
+      </div>
+    </div>
+  {% endfor %}
+{%- endmacro %}
+
+{%- macro promoted_header(formatted_query, promoted_count, promoted_count_formatted, top_of_tree, tree) -%}
+    <p class="promoted-see-more">
+        <a href="{{ www_root }}/{{ tree }}/search?q=path:{{ formatted_query }}">{{ promoted_count_formatted }}
+            path {{ 'result' if promoted_count == 1 else 'results' }}</a> from the <a
```

Comment:
I don't think we need to include "from the foo tree" here after moving promoted results to the right. The main results that need nicely and are the first thing the eye sees.